docs/prompt_hacking/injection.md

```diff
@@ -36,6 +36,29 @@ The reason this works is that `remoteli.io` takes a user's tweet and concatenate
 with their own prompt to form the final prompt that they pass into a LLM. This means that
 any text the Twitter user injects into their tweet will be passed into the LLM.
 
+## Token Smuggling
+
+The term token smuggling refers to supplying the model with otherwise banned words. If you try to enter any of these words in the prompt directly, the model will not work. If you use code, and pass the words in pieces. When the model reconstructs the code from the previous prompt, it forces the model to complete the sentence. We can again mask it as a coding task (called autoregressive generation), and fool it into generating a sample output of the code, where it is originally working on malicious prompt. The smuggled tokens can virtually model to create any scenario - (@Ninkat2023TokensmugglingDef)(@vaibhavk972023Tokensmuggling)(@Ninkat2023TokensmugglingDef)
+
+### Masked Language Modeling
+
+The task of masking tokens in a sequence with a masking token and directing the model to fill that mask with an appropriate token is known as masked language modeling. This allows the model to focus on both the right and left contexts (tokens on the right side of the mask) (tokens on the left of the mask).
```

Comment:
Why is this here? Same question with Autoregressive language modeling and Autoencoder (AE) Language Model